docs/source/developers/release.rst

```diff
@@ -61,7 +61,6 @@ generated properly.
         mvn clean install -Papache-release
 
     - Have the build requirements for cpp and c_glib installed.
-    - Set the ``JIRA_USERNAME`` and ``JIRA_PASSWORD`` environment variables
```

Comment:
I can't see any reference to `JIRA_USERNAME` nor `JIRA_PASSWORD` anywhere on our repo. I think those are not used.